pymodbus/constants.py

```diff
@@ -44,12 +45,8 @@ class ModbusStatus:  # pylint: disable=too-few-public-methods
     SlaveOn = 0xFF
     SlaveOff = 0x00
 
-    def __init__(self):
-        """Prohibit objects."""
-        raise RuntimeError(INTERNAL_ERROR)
 
-
-class Endian:  # pylint: disable=too-few-public-methods
```

Comment:
This should be replaced by the system constant.

Reply:
Not sure I understood, do you mean `sys.byteorder`?